docs/sdk/01-examples/02-multisig-client/09-get-proposals.md

```diff
@@ -29,11 +29,12 @@ const queryParams: IProposalQueryParams = {
   direction: SortDirection.ASC, // optional. otherwise, DESC
   sortBy: ProposalSortBy.CREATED_AT, //optional. otherwise, NAME, VOTES (POPULARITY coming soon)
   status: ProposalStatus.ACTIVE, // optional. otherwise, PENDING, SUCCEEDED, EXECUTED, DEFEATED
-  daoAddressOrEns: "0x1234348529348570294650287698237520938574284357" // or my-dao.dao.eth
+  daoAddressOrEns: "0x1234348529348570294650287698237520938574284357", // or my-dao.dao.eth
 };
 
-const multisigProposals: MultisigProposalListItem[] = await multisigClient.methods.getProposals(queryParams);
-console.log({ multisigProposals });
+const multisigProposals: MultisigProposalListItem[] = await multisigClient
+  .methods.getProposals(queryParams);
+console.log(multisigProposals);
```

Comment:
Same here - 

Let's use the stanrdard ({ multisigProposals }) so the console log is explicit about what it is returning.